Lib/threading.py

```diff
@@ -1698,6 +1697,9 @@ def _after_fork():
         # by threading.Thread. For example, a thread spawned
         # by thread.start_new_thread().
         current = _MainThread()
+    else:
+        if isinstance(current, _DummyThread):
+            current = _MainThread()
```

Comment:
Please add a comment explaining that creating _MainThread() instance overrides `_active[get_ident()]`.